Code/Legacy/CryCommon/CryThreadImpl_pthreads.h

```diff
@@ -14,13 +14,7 @@
 
 #include "CryThread_pthreads.h"
 
-#if PLATFORM_SUPPORTS_THREADLOCAL
-THREADLOCAL CrySimpleThreadSelf
-* CrySimpleThreadSelf::m_Self = NULL;
-#else
-TLS_DEFINE(CrySimpleThreadSelf*, g_CrySimpleThreadSelf)
-#endif
```

Comment:
These thread local changes aren't necessary, the whole CrySimpleThread concept is getting deleted in my next Cry-Threading cleanup pass.  As is those TLS macros in platform.h

Reply:
ok, should be an easy merge if both are deleted, there were some cases where the `THREADLOCAL` was used in other files. Replaced it with the AZ alternative